query/aggregator.go

```diff
@@ -162,13 +218,13 @@ func applyDiv(a, b, c *types.Val) error {
 	switch vBase {
 	case INT:
 		if b.Value.(int64) == 0 {
-			return errors.Errorf("Division by zero")
+			return ErrorDivisionByZero
 		}
 		c.Value = a.Value.(int64) / b.Value.(int64)
 
 	case FLOAT:
 		if b.Value.(float64) == 0 {
-			return errors.Errorf("Division by zero")
+			return ErrorDivisionByZero
```

Comment:
IEEE 754 says that floating point values divided by zero should return infinity. Clarify what we want to do here.